CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.  The format
 Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+## 0.2.3 10-30-2020
+### Changed
+- Change `DateTimeSeconds` type for information elements to `uint32` following RFC7011. (#59, @zyiou)
+- Change PEN number for Antrea to '56506' (assigned by IANA). (#60, @zyiou)
+### Added
+- Improve test coverage for IPv6 support. (#61, @zyiou)
```

Comment:
Could you please add about the support for IPv6 clusterIP in this patch release?

Reply:
Updated. Thanks!